tools/mempool-pusher.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+
+exit_error() {
+   printf "$1" >&2
+   exit 1
+}
+
+for cmd in sqlite3 curl xxd grep cut jq date; do
+   which $cmd >/dev/null 2>&1 || exit_error "Missing command: $cmd"
+done
+
+if [ $(echo ${BASH_VERSION} | cut -d '.' -f 1) -lt 4 ]; then
+   exit_error "This script requires Bash 4.x or higher"
+fi
+
+set -uo pipefail
+
+DEBUG=1
+
+log() {
+   printf >&2 "%s\n" "$1"
+}
+
+debug() {
+   if [ $DEBUG -ne 0 ]; then
+      log "$1"
+   fi
+}
+
+# args:
+# $1 mempool path
+# $2 minimum accept time
```

Comment:
Looks great! Any chance we can add a commented example for each function?

Reply:
Added in 23a8adcf6c67cb76d6aaccf00516d7bc272ce11a

Reply:
Thank you!!